pkg/envoy/accesslog/v2/field_operator_test.go

```diff
@@ -6,10 +6,10 @@ import (
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/ginkgo/extensions/table"
 	. "github.com/onsi/gomega"
+	"google.golang.org/protobuf/types/known/durationpb"
```

Comment:
Why use a different duration wrapper package here, when everywhere else we use `ptypes/duration`? We should be consistent unless there's a special reason for it.